README.md

```diff
@@ -0,0 +1,236 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🖺 Entity Relationship Diagram](#er-diagram)
+  - [📽️ Project Documentation](#pro-doc)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My-Exp <a name="about-project"></a>
+![App-gif](./app/assets/images/my-exp.gif)
+
+**My-Exp** is a mobile web application that allows the user to:  
+ - register and log in, so that the data is private to them.
+ - Ceate categories for thier expenses.
+ - record transactions associated with a category.
+ - see the money spent on each category.
+ - see the total money spent on all categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+</details>
+
+<details>
+  <summary>Ruby on Rails</summary>
+</details>
+
+<details>
+  <summary> Javascript </summary>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Authentication - User must be authenticated**
+- **Users can track expenses category-wise**
+- **Users can see the total money spent on all categories**
+- **Users expense data is private to them**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Project Documentation -->
+## 📽️ Project Doocumentation <a name="pro-doc"></a>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [My-Exp](https://my-expenses-3hoz.onrender.com) 
+
+- [video](https://www.loom.com/share/d17398b07917461e8659276a5500acc3)
```

Comment:
[REQUIRED] Good job updating your video but now it is too long. One of the requirements is that your video should not exceed 5 minutes.  I think this is because you explained a large bulk of your code. All you need to do is to show us a presentation of your project, your live demo (the most important features), and just  a piece of code that you are impressed with not the entirety of your code.